.drone.env

```diff
@@ -1,3 +1,3 @@
 # The test runner source for UI tests
-WEB_COMMITID=e518f4c2d19ccd233666e7dd46ad8f38ecdbdf00
-WEB_BRANCH=master
+WEB_COMMITID=67ced24b3a6f06b767ff8111537b4a0b85c40993
+WEB_BRANCH=changeRespCode
```

Comment:
This is temporary because we need a green ocis version to get a green web version, right?

Reply:
You are rigth